fake-api.js

```diff
@@ -49,6 +130,20 @@ app.get('/auth', (req, res) => {
   res.status(200).end();
 });
 
+app.use('/', express.static('test/fake-api-fixtures', { index: 'index.json' }));
+
+app.post('/*', (req, res) => {
```

Comment:
I don't think we need these if we're going to have specific routes for each data type